src/resources/ResourceSnapshots/ResourceSnapshotsInterfaces.ts

```diff
@@ -52,6 +52,10 @@ export interface PushSnapshotOptions {
     developerNotes?: string;
 }
 
+export interface CreateFromFileOptions {
+    developerNotes?: string;
```

Comment:
`snapshotFileType` should be in there no?

Reply:
No because we don't pass it from the Admin UI, we compute it in the function